doc/source/plans.rst

```diff
@@ -109,6 +109,37 @@ Examples:
     RE.subscribe(bec)
     RE(count([noisy_det], num=5))
 
+.. note::
+
+   Why doesn't :func:`count` have an ``exposure_time`` parameter?
+
+   Modern CCD detectors typically parametrize exposure time with *multiple*
+   parameters. There is no one "exposure time" that can be applied to all
+   detectors.
+
+   Additionally, counts with multiple detectors as in ``count([det1, det2]))``
+   would need to provide a separate exposure time for each detector in the
+   general case, which would grow wordy.
+
+   We recommend either setting the time-related parameter(s) in advance:
+
+   .. code-block:: python
```

Comment:
I agree with [Tom](https://github.com/bluesky/bluesky/pull/1223#pullrequestreview-255874096).  Suggest replacing lines 124-128 with this:

```
We recommend either setting the time-related parameter(s) in advance.
For interactive use:

   .. code-block:: python

      det.exposure_time.set(3)

From a plan:

   .. code-block:: python

      yield from bluesky.plan_stubs.mv(det.exposure_time, 3)
```